datauploader/tasks.py

```diff
@@ -131,3 +131,10 @@ def upload_file_to_oh(oh_member, filepath, metadata):
 
     logger.debug('Upload done: "{}" for member {}.'.format(
             os.path.basename(filepath), oh_member.oh_id))
+
+
+@shared_task
+def make_request_respectful_get(url, realms, **kwargs):
+    r = settings.rr.get(url=url, realms=realms, **kwargs)
```

Comment:
If you don't want to have to use `settings.rr.get` you can (in the top of the file) do `from oh-fitbit-integration.settings import rr` and then just do `rr.get`

Reply:
I don't think you can do that because in that case it would be using `oh-fitbit-integration` as a variable, and variable names in python can't contain hyphens... maybe I am wrong, but I couldn't get it to work